cmd/notary-server/config.go

```diff
@@ -25,7 +25,7 @@ import (
 	_ "github.com/lib/pq"
 	"github.com/spf13/viper"
 	"golang.org/x/net/context"
-	gorethink "gopkg.in/dancannon/gorethink.v2"
```

Comment:
I found out about it recently, because `goimports` keeps doing the wrong thing when I save (https://github.com/joefitzgerald/go-plus/issues/52), but apparently if there's a `.` or `-` or some other special character, everything after that gets ignored for naming?